os/se/ameba/Kconfig

```diff
@@ -44,3 +44,20 @@ config HW_SE_STORAGE
 	---help---
 		SE supports Secure Stroage
 
+config SE_FACTORY_CERT_ADDRESS
+	hex "Factoty Cert Address"
+	default 0x1C0000
```

Comment:
In defconfig, you set different address. Is there any specific reason to set differently?

Reply:
This address is still under discussion, currently this is a temporary location. Maybe next version I will remove it.

Reply:
Since it will be place after the bootloader as a fix location, so this config setting will be remove.